src/components/AppNavBar/AppNavBarPresentation.js

```diff
@@ -102,11 +104,54 @@ function AppNavBarPresentation({ isHome = false, login, logout }) {
                 {getIcon('FVLogo', 'fill-current h-10')}
               </span>
             </a>
-            <div className="w-16 text-white text-sm leading-tight text-center mr-2">
+            <button
+              className="w-16 text-white text-sm leading-tight text-center mr-2"
+              onClick={() => setBetaModalOpen(true)}
+              type="button"
+            >
               Beta Version
-            </div>
+            </button>
           </div>
 
+          <Modal.Presentation
+            isOpen={betaModalOpen}
+            closeHandler={() => setBetaModalOpen(false)}
+          >
+            <div
+              data-testid="BetaModalContent"
+              className="bg-white rounded-lg shadow-lg px-6 py-32"
+            >
+              <h2 className="text-3xl font-bold tracking-tight text-fv-charcoal sm:text-4xl">
+                Welcome to the new FirstVoices!
+              </h2>
+              <p className="mx-auto mt-3 md:mt-6 max-w-xl text-lg leading-8 text-fv-charcoal-light">
+                The beta version of FirstVoices v3 is now live.
+                <br /> If you are looking for the old version of FirstVoices,{' '}
+                <br />
+                please click{' '}
+                <a
+                  href="www.archive.firstvoices.com"
+                  className="text-blue-600 visited:text-purple-600 underline underline-offset-2"
+                >
+                  here
+                </a>
+                {/*
+                 */}
+                . Having issues with the new version? <br />
+                Please contact us{' '}
+                <a
+                  href="mailto:hello@firstvoices.com"
```

Comment:
(Optional) If the link to the archived firstvoices or the email id used here are used any place else, they can be put into constants and those constants can be used instead.

Reply:
Good point.  It doesn't seem that it appears in this form. It's only written out as text in the code so I will leave this as is right now.